public_html/wp-content/plugins/wcpt/stubs/page/offline.php

```diff
@@ -0,0 +1,23 @@
+<!-- wp:paragraph {"customBackgroundColor":"#eeeeee"} -->
+<p style="background-color:#eeeeee" class="has-background"><?php esc_html_e( "Organizers note: Update this page with some basic information about your WordCamp. It will be stored in site visitor's browsers, and automatically shown if they try to visit the site without an internet connection.", 'wordcamporg' ); ?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><?php esc_html_e( "This page couldn't be loaded because you appear to be offline. Please try again once you have a network connection.", 'wordcamporg' ); ?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><?php esc_html_e( 'In the mean time, hopefully this information is useful:', 'wordcamporg' ); ?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading -->
+<h2><?php esc_html_e( 'Location', 'wordcamporg' ); ?></h2>
+<!-- /wp:heading -->
+
+<!-- wp:heading -->
+<h2><?php esc_html_e( 'Schedule', 'wordcamporg' ); ?></h2>
+<!-- /wp:heading -->
+
+<!-- wp:shortcode -->
+[schedule date="YYYY-MM-DD" tracks="example-track,another-example-track,yet-another-example-track"]
```

Comment:
```suggestion
[schedule]
```

`MAYBE` - Since the organizers aren't likely to revisit this page when they add/remove/edit tracks, it might be safest to leave off all the options and just display everything by default.